src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Account/Pages/ExternalLogin.razor

```diff
@@ -127,6 +132,11 @@
 
     private async Task OnValidSubmitAsync()
     {
+        if (externalLoginInfo is null)
+        {
+            RedirectManager.RedirectToWithStatus("Account/Login", "Error loading external login information.", HttpContext);
```

Comment:
Nit:

```suggestion
            RedirectManager.RedirectToWithStatus("Account/Login", "Error loading external login information during confirmation.", HttpContext);
```

To match [the message we use in Razor Pages](https://github.com/dotnet/aspnetcore/blob/15a4a77ef43d65dd19ebcc925c0aefcedeb3b80e/src/Identity/UI/src/Areas/Identity/Pages/V5/Account/ExternalLogin.cshtml.cs#L174-L179).